test/factree/aggregate_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+describe Factree::Aggregate do
```

Comment:
Yes minitest! Minitest::Spec looks 'close enough' to RSpec. Internally it's implemented in a more sane fashion. Also Minitest deliberately does not implement some of RSpec's craziest features: `shared_context`, `shared_examples` and friends. This makes me happy.